easybuild/easyconfigs/d/dotNET-SDK/dotNET-SDK-3.1.300-GCCcore-10.1.0.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	MIT
+#
+# Notes::	requires glibc v2.14 or higher
+##
+
+easyblock = 'Tarball'
+
+name = 'dotNET-SDK'
+version = '3.1.300'
+
+homepage = 'https://www.microsoft.com/net/'
+description = """.NET is a free, cross-platform, open source developer platform for building many different types
+ of applications."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.1.0'}
+
+source_urls = ['https://download.visualstudio.microsoft.com/download/pr/cdb98822-bf8f-4f11-a5a4-baa8d8ad66d0/b696896552ca7768a7189f775e220d2e']
+sources = ['dotnet-sdk-%(version)s-rhel.6-x64.tar.gz']
+checksums = ['4bd844dec97011f7dee754d37d7c7b12c7f843a4e5484dabc77c8737075c8937']
+
+sanity_check_paths = {
+    'files': ['dotnet'],
+    'dirs': ['shared/Microsoft.NETCore.App', 'shared/Microsoft.AspNetCore.App'],
+}
+
+modextrapaths = {'PATH': ['']}
```

Comment:
I think `DOTNET_ROOT` should be set as well ( based on what i saw on https://docs.microsoft.com/en-us/dotnet/core/install/sdk?pivots=os-linux#download-and-manually-install )